packages/relay/test/report.test.ts

```diff
@@ -139,12 +139,45 @@ describe('POST /api/report', function () {
         await stopServer('report', snapshot, sync, express)
     })
 
+    it('should fail to create a report on commentwith empty postId', async function () {
```

Comment:
nit: space between "comment" and "with", also this test is actually testing that report on POST shouldn't have "postId" included?

Reply:
I have implemented postId in the schema as required to enforce field validation. You can see the implementation here:

https://github.com/social-tw/social-tw-website/blob/e4c3c729977afa9c022d558af3ff31df839b04f3/packages/relay/src/db/schema.ts#L150

Additionally, in the ReportService on the frontend, we ensure that if postId is undefined (for example, during a post report), we pass an empty string as postId. You can view the relevant code here:

https://github.com/social-tw/social-tw-website/blob/e4c3c729977afa9c022d558af3ff31df839b04f3/packages/frontend/src/features/core/services/ReportService/ReportService.ts#L70